README.md

```diff
@@ -1 +1,50 @@
 # HangOps Code of Conduct
+
+To continue growing a safe, welcoming, and fair community, all HangOps members are required to comply with the following code of conduct. HangOps admins are responsible for ensuring that all community members follow this code within HangOps spaces.
+
+## The Short Version
+
+Be respectful of other people; respectfully ask people to stop if you are bothered; and if you can’t resolve an issue contact the admins. If you are unable to follow our Community Code of Conduct problem, it will be apparent and you may be asked to leave HangOps.
+
+## The Long Version
+
+### Respect
+
+HangOps is a community of people who care about running production systems reliably. We have diverse backgrounds, skills, talents, experiences, cultures, and opinions. We are an inclusive environment and we treat all individuals respectfully.
```

Comment:
This isn't an oxford comma case, although I am a big fan of the oxford comma.

This case is two independent clauses joined by a conjunction.

* We are an inclusive environment.
* and
* We treat all individuals respectfully.

Since both are complete sentences on their own (subject + verb, etc), if you want to join them together with a coordinating conjunction ("and"), it has been my experience that you join them with a comma: `clause1, and clause2`.

References: http://grammar.ccc.commnet.edu/Grammar/indep_clauses.htm and https://owl.english.purdue.edu/owl/resource/598/01/

If you want to omit a comma, you can remove the word `we` and the subject is assumed the same:

"We are an inclusive environment and treat all individuals respectfully"

or

"We are an inclusive environment, and we treat all individuals respectfully."

----

Though, if you want to rephrase to emphasis responsibility of the community towards itself, I would consider rephrasing it: "The community provides an inclusive environment by treating all individuals respectfully"